### PR TITLE
Models

### PR DESCRIPTION
**Pull request by elreimundo**
adds validations and associations for team, league and user classes

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
-class User
+class User < ActiveRecord::Base
+  has_many :teams
+  has_many :leagues through :teams
+  validates :username, :email, uniqueness: true
+  validates :firstname, :lastname, :username, :email, :password, presence: true
 end
```
Comment (zspencer) [inline app/models/user.rb:7]: Y U NO TEST YOUR VALIDATIONS?!

**Comment by raorao** [comment]
this should be tested.

File: app/models/user.rb (hunk 1 above)
Comment (elreimundo) [inline app/models/user.rb:7]: I thought "TDD" meant "To do: develop."

**Comment by nscricco** [comment]
witty comment

File: logfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+LOG:  database system was shut down at 2013-09-30 16:26:11 PDT
+LOG:  database system is ready to accept connections
+LOG:  autovacuum launcher started
+FATAL:  database "fantasy_test" does not exist
+ERROR:  relation "politicians" does not exist at character 29
+STATEMENT:  SELECT "politicians".* FROM "politicians"
+ERROR:  relation "politicians" does not exist at character 29
+STATEMENT:  SELECT "politicians".* FROM "politicians"
```
Comment (waneka) [inline logfile:8]: [David] Do we really need the logfile?

Comment (zspencer) [inline logfile:8]: I would recommend ignoring the logfile.

**Pull request merged**